### DISCUSSION TITLE
Contribution graph is not showing for commits in same repository

### DISCUSSION THREAD

**Discussion by Nidhin-mohan** [community/community#40513] [Repositories]
Contribution graphs are only showing when there is new repository is created  . Not showing contribution graph for commits that are on the same repo. I am pushing my code from VScode so  I don't have to add my username or password(key) is this the reason for not showing commits??

**Comment by radar-blip** [comment]
That could be the reason since your commits are tied to your email.
Also, check this in your settings:

![commits](https://user-images.githubusercontent.com/39196767/204964550-9e169e55-94f3-4815-8a63-aa5c8ed7c2d4.png)
